src/Sarif.Driver/Sdk/AnalyzeOptionsBase.cs

```diff
@@ -109,5 +109,10 @@ public abstract class AnalyzeOptionsBase : CommonOptionsBase
             "baseline",
             HelpText = "A Sarif file to be used as baseline")]
         public string BaselineSarifFile { get; set; }
+
+        [Option(
+            "exportUri",
```

Comment:
>exportUri [](http://example.com/codeflow?start=13&length=9)

Suggest you call this 'postUri'. Makes sense, right? :) Generally, we don't 'export' to web endpoints, we publish or post.

Reply:
thanks for the suggestion :)